### ISSUE TITLE
Embeddable examples for Orange Design System portal

### ISSUE THREAD

**Issue by julien-deramond** [Orange-OpenSource/ods-charts#108]
### Context

Boosted provides components for web developers whose the design is described in Orange Design System Design System Manager: https://system.design.orange.com

For the buttons for instance, at https://system.design.orange.com/0c1af118d/p/278ebc-buttons-standard/b/0465a6, in the "Develop" tab, we can see the rendering of the buttons available in Boosted.

It's embedded here via a specific Storybook page: https://boosted.orange.com/storybook/?path=/docs/components-buttons--docs

### Description

We'd like to do the same thing for ODS Charts. Zeroheight allows us to embed elements from Storybook, but also from whatever website as iframes. The idea here would be to reproduce the same examples provided in the DSM in some `test/zeroheight/*.html` independent pages and embed them into the DSM.

**Comment by jacques-lebourgeois** [comment]
@julien-deramond , do you have a pointer to someone who would be able to make such an integration and specify the necessary evolutions on the ODS Charts library side?

**Comment by julien-deramond** [comment]
Yep, I'll send you the contact internally. It might be changed with the new Orange Unified Design System.

**Comment by jacques-lebourgeois** [comment]
@louismaximepiton , I think this is the need you had to have the ODS Charts display embedded in OUDS documentation
